module/EtRobocon2024.cpp

```diff
@@ -53,6 +53,24 @@ void EtRobocon2024::start()
   bool isLeftEdge = true;
   Calibrator calibrator;
 
+  //　　　走行ログファイルを取得する場合
+  if(shouldGetRunLog == true) {
```

Comment:
```suggestion
  if(shouldGetRunLog) {
```
下のと合わせるならこう？　上と下どっちに合わせるかは任せます。

Reply:
レビューありがとうございます。下の記法に統一しました。